skillsmap/src/components/Carousel.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import { connect } from 'react-redux';
+
+import { SkillsMapState } from '../store/reducer';
+import { Item, CarouselItem } from './CarouselItem';
+import { dispatchChangeSelectedItem } from '../actions/dispatch';
+
+import '../styles/carousel.css'
+
+interface CarouselProps {
+    items: Item[];
+    selectedItem?: string;
+    itemTemplate?: any;
```

Comment:
Could this use a functional component instead of any? i.e. `(props: ItemProps) => JSX.Element`